src/OrbitQt/CaptureOptionsDialog.ui

```diff
@@ -61,6 +61,79 @@
        </layout>
       </widget>
      </item>
+     <item>
+      <widget class="QGroupBox" name="groupBox_4">
+       <property name="title">
+        <string>Memory tracing</string>
+       </property>
+       <layout class="QVBoxLayout" name="verticalLayout_5">
+        <item>
+         <widget class="QCheckBox" name="collectMemoryInfoCheckBox">
+          <property name="text">
+           <string>Collect system-wide memory usage information</string>
+          </property>
+         </widget>
+        </item>
+        <item>
+         <layout class="QHBoxLayout" name="memoryHorizontalLayout">
+          <item>
+           <widget class="QLabel" name="memorySamplingPeriodNsLabel">
+            <property name="text">
+             <string>Sampling period (ns):</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLineEdit" name="memorySamplingPeriodNsLineEdit">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+            <property name="inputMethodHints">
+             <set>Qt::ImhDigitsOnly</set>
+            </property>
+            <property name="text">
+             <string>100000000</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <spacer name="horizontalSpacer_2">
+            <property name="orientation">
+             <enum>Qt::Horizontal</enum>
+            </property>
+            <property name="sizeHint" stdset="0">
+             <size>
+              <width>60</width>
+              <height>20</height>
+             </size>
+            </property>
+           </spacer>
+          </item>
+          <item>
+           <widget class="QLabel" name="memoryWarningThresholdKbLabel">
+            <property name="text">
+             <string>Warning threshold (kb):</string>
+            </property>
+           </widget>
+          </item>
+          <item>
+           <widget class="QLineEdit" name="memoryWarningThresholdKbLineEdit">
+            <property name="enabled">
+             <bool>false</bool>
+            </property>
+            <property name="inputMethodHints">
+             <set>Qt::ImhDigitsOnly</set>
+            </property>
+            <property name="text">
+             <string>8388608</string>
```

Comment:
Can we write a comment there? "#8GB" or "8 x 1024 x 1024"

Reply:
Good idea. I added a tooltip "The default value is 8388608 KB, i.e., 8GB." for this.

Reply:
I was thinking more in adding a comment to have in the .ui file, but your change probably is better :)